### PR TITLE
feat: change MessageModel columns

### PR DESCRIPTION
**Pull request by penguinuux**
Removed columns title and text from "messages" table;
Set column date a default value.

### CODE REVIEW

File: app/models/message_model.py [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
 from app.configs.database import db
 
 
 @dataclass
 class MessageModel(db.Model):
-    title: str
-    text: str
-    date: str
-
     __tablename__ = "messages"
```
Comment (marcelosj3) [inline app/models/message_model.py]: Shouldn't we change the name to `message_logs` as well?

File: app/models/message_model.py [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
 
     id = Column(Integer, primary_key=True)
-    title = Column(String, nullable=False)
-    text = Column(String, nullable=False)
-    date = Column(DateTime, nullable=False)
+    date = Column(DateTime, nullable=False, default=datetime.now())
```
Comment (marcelosj3) [inline app/models/message_model.py:14]: Just asking, but the relationships of this table were made in other tables, right?

**Review by marcelosj3 — APPROVED** [review]

File: app/models/message_model.py (hunk 2 above)
Comment (penguinuux) [reply app/models/message_model.py:14]: Right. The relationship are in the UserModel.

**Review by JOSUE-ASV — APPROVED** [review]

File: app/models/message_model.py (hunk 1 above)
Comment (penguinuux) [reply app/models/message_model.py]: Done.

**Review by marcelosj3 — APPROVED** [review]

**Pull request merged**